blueprints/codes/eurocode/nen_en_1992_1_1_c2_2011/chapter_4_durability_and_cover/table_4_3.py

```diff
@@ -0,0 +1,173 @@
+"""Module for the concrete structural class
+according to Table 4.3 from NEN-EN 1992-1-1+C2:2011: Chapter 4 - Durability and cover to reinforcement.
+"""
+
+from __future__ import annotations
+
+from typing_extensions import Self
+
+from blueprints.codes.eurocode.nen_en_1992_1_1_c2_2011 import NEN_EN_1992_1_1_C2_2011
+from blueprints.codes.eurocode.nen_en_1992_1_1_c2_2011.chapter_4_durability_and_cover.table_4_1 import (
+    Carbonation,
+    Chloride,
+    ChlorideSeawater,
+    ExposureClasses,
+)
+from blueprints.codes.eurocode.structural_class import AbstractConcreteStructuralClassCalculator, ConcreteStructuralClassBase
+from blueprints.materials.concrete import ConcreteMaterial, ConcreteStrengthClass
+from blueprints.type_alias import YEARS
+
+"""Design working life in years as defined in table 4.3 of NEN-EN 1992-1-1+C2:2011."""
+DESIGN_WORKING_LIFE_DEFAULT = 50
+DESIGN_WORKING_LIFE_75 = 75
+DESIGN_WORKING_LIFE_100 = 100
+
+
+class ConcreteStructuralClassCalculator(AbstractConcreteStructuralClassCalculator):
```

Comment:
Question: given that this is the implementation of Table 4.3 should be that reflected in the name just like the formulas?

Reply:
That's a fair point. I've changed the name to Table4Dot3ConcreteStructuralClass